README.md

```diff
@@ -24,21 +24,50 @@ flutter pub add volt
 
 ## Usage
 
+#### Query
+
 ```dart
 VoltQuery<Photo> photoQuery(String id) => VoltQuery(
-      queryKey: ["photo", id],
-      queryFn: () => fetch("https://jsonplaceholder.typicode.com/photos/$id"),
+      queryKey: ['photo', id],
+      queryFn: () => fetch('https://jsonplaceholder.typicode.com/photos/$id'),
       select: Photo.fromJson,
     );
 
 Widget build(BuildContext context) {
-  final photo = useQuery(photoQuery("1"));
+  final photo = useQuery(photoQuery('1'));
+
+  return photo == null ? CircularProgressIndicator() : Text('Photo: ${photo.title}');
```

Comment:
Just checked this example. Isn't useQuery supposed to return an object that contains the data and the query state etc?